package-lock.json

```diff
@@ -4509,6 +4509,16 @@
       "integrity": "sha512-nAihlQsYGyc5Bwq6+EsubvANYGExeJKHDO3RjnvwU042fawQTQfM3Kxn7IHUXQOz4bzfwsGYYHGSvXyW4zOGLg==",
       "dev": true
     },
+    "bindings": {
```

Comment:
Revert changes

Reply:
@DanielFran the current file did not listed in our pr.

Reply:
@vkhsunny 
![image](https://user-images.githubusercontent.com/3706415/102919477-d8e01980-4480-11eb-8ace-40f96cc3e682.png)